src/heatmap.ts

```diff
@@ -16,7 +16,7 @@ import {IVector} from 'phovea_core/src/vector';
 import {toSelectOperation, defaultSelectionType} from 'phovea_core/src/idtype';
 
 
-function defaultColor(value: any) {
+export function defaultColor(value: any) {
```

Comment:
Why is it here casted to `any`, but in `defaultDomain()` not. Can we remove it here or specify the type? Would be helpful for your test, too.